schema.graphql

```diff
@@ -221,8 +247,8 @@ type PositionChanged @entity {
     referralCode: ReferralCode
 }
 
-type PositionClosed @entity {
-    "format: <tx hash>-<tx log index>"
+type PositionClosed @entity(immutable: true) {
```

Comment:
A random question. Is `PositionClosed` subset of `PositionChanged`? If so, do we still need `PositionClosed`?

Reply:
No, they're different. `PositionClosed` is emitted when calling `ClearingHouse.quitMarket()`. It's for shutting down a market (ex: `LUNA`).